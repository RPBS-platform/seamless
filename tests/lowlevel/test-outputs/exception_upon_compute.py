Status: {
  "result": "undefined",
  "tf": "error"
}

**************************************************
* STDOUT:                                        *
**************************************************
TEST
**************************************************

Traceback (most recent call last):
  File "/seamless/seamless/core/execute.py", line 87, in _execute
    exec_code(code, identifier, namespace, inputs, output_name)
  File "/seamless/seamless/core/cached_compile.py", line 58, in exec_code
    exec(code_obj, namespace)
  File "Seamless transformer: .tf", line 1, in <module>
    print('TEST'); raise Exception(a)
Exception: 1

