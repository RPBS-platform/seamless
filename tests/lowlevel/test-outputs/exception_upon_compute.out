Status: {
  "tf": "error"
}

*************************************************
* Standard output
*************************************************
TEST

*************************************************
Traceback (most recent call last):
  File "/seamless/seamless/core/execute.py", line 67, in _execute
    exec_code(
  File "/seamless/seamless/core/cached_compile.py", line 62, in exec_code
    exec(code_obj, namespace)
  File "Seamless transformer: .tf", line 1, in <module>
    print('TEST'); raise Exception(a)
Exception: 1

Status: OK
None
OK
